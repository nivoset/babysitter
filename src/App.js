import React from 'react';
import { store } from "./store";
import { Provider } from "react-redux";
import { BabysitterController } from "./controllers/Babysitter/index.js"
import { MuiPickersUtilsProvider } from "@material-ui/pickers";
import DateFnsUtils from '@date-io/date-fns';

function App() {
  return (
    <MuiPickersUtilsProvider utils={DateFnsUtils}>
      <Provider store={store}>
        <BabysitterController />
      </Provider>
    </MuiPickersUtilsProvider>
  );
}

export default App;
