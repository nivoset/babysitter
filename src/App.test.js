import React from 'react';
import { create } from 'react-test-renderer';
import App from './App';

test('renders learn react link', () => {
  const component = create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
