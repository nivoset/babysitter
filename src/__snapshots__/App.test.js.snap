// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="display-items"
>
  <div
    id="start-date"
    title="Start Time"
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
      format="hh:mm a"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          readOnly={true}
          required={false}
          type="text"
          value="05:00 PM"
        />
      </div>
      
    </div>
  </div>
  <div
    id="bedtime"
    title="Bed Time"
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
      format="hh:mm a"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          readOnly={true}
          required={false}
          type="text"
          value="09:00 PM"
        />
      </div>
      
    </div>
  </div>
  <div
    id="end-date"
    title="End Time"
  >
    <div
      className="MuiFormControl-root MuiTextField-root"
      format="hh:mm a"
      onClick={[Function]}
      onKeyDown={[Function]}
    >
      <div
        className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          readOnly={true}
          required={false}
          type="text"
          value="10:00 PM"
        />
      </div>
      
    </div>
  </div>
  <div
    id="display"
    title="Display wage earned"
  >
    wages earned= $56
  </div>
</div>
`;
